src/lib/ui/thermostat/ClimateSettingStatus.tsx

```diff
@@ -6,7 +6,8 @@ import { ThermostatHeatCoolIcon } from 'lib/icons/ThermostatHeatCool.js'
 import { ThermostatOffIcon } from 'lib/icons/ThermostatOff.js'
 
 interface ClimateSettingStatusProps {
-  climateSetting: ClimateSetting
+  climateSetting: ClimateSetting &
```

Comment:
Can you explain the need for this complex type? I'm not sure I understand what is happening here.

Reply:
The `ClimateSetting` type requires `manual_override_allowed` be defined, but this component doesn't need that property at all. I don't want to require it, nor do I want to error if it is provided. 

`Omit<ClimateSetting, "manual_override_allowed">`, which is what I tried initially, will not typecheck if `manual_override_allowed` is defined. 

<img width="418" alt="Screenshot 2023-08-14 at 4 07 35 PM" src="https://github.com/seamapi/react/assets/11382084/0d587946-231a-4433-b7f4-a4c496f23ef6">

Reply:
It's possible I'm overthinking here and should just use `Partial<ClimateSetting>`

Reply:
https://github.com/seamapi/react/pull/348/commits/993fe636f4ba6d9cafa227d1a246853ac66045fe